src/main/java/de/retest/persistence/migration/XmlMigrator.java

```diff
@@ -47,7 +47,11 @@ public static NamedBufferedInputStream tryToMigrate( final XmlVersionCheckResult
 			for ( int i = 0; i < transformers.size(); i++ ) {
 				final boolean notFinalInputStream = i < transformers.size() - 1;
 				final InputStream previousInputStream = finalInputStream;
-				finalInputStream = transformers.get( i ).transform( finalInputStream );
+				try {
+					finalInputStream = transformers.get( i ).transform( finalInputStream );
+				} catch ( final IOException e ) {
+					throw new RuntimeException( e );
```

Comment:
In the below try-catch we just log the error. Which method is preferred since we exit the execution here? In either case, I would log the error.